application/config/internal.php

```diff
@@ -118,7 +118,7 @@
         // These are defaults and are later overwritten in LSYii_Application by a path based on config tempdir/tempurl
         'assetManager' => array(
             'excludeFiles' => array("config.xml", "assessment.pstpl", "clearall.pstpl",  "completed.pstpl",  "endgroup.pstpl",  "endpage.pstpl",  "groupdescription.pstpl",  "load.pstpl",  "navigator.pstpl",  "printanswers.pstpl",  "print_group.pstpl",  "print_question.pstpl",  "print_survey.pstpl",  "privacy.pstpl",  "question.pstpl",  "register.pstpl",  "save.pstpl",  "startgroup.pstpl",  "startpage.pstpl",  "surveylist.pstpl",  "survey.pstpl",  "welcome.pstpl" ),
-            'baseUrl' => '/tmp/assets',
+            'baseUrl' => '/lime-dev/tmp/assets',
```

Comment:
Fixed in https://github.com/LimeSurvey/LimeSurvey/commit/71685ac145af96adfbe85d9eb22d36488f80516e , thanks for report